api/services/responses/SuccessResponseService.cs

```diff
@@ -6,9 +6,9 @@
 
 namespace LiteralLifeChurch.LiveStreamingApi.services.responses
 {
-    class SuccessResponseService<Output> : IResponseService where Output : IOutputModel
+    public static class SuccessResponseService
     {
-        public HttpResponseMessage CreateResponse(Output response, HttpStatusCode statusCode = HttpStatusCode.OK)
+        public static HttpResponseMessage CreateResponse<Output>(Output response, HttpStatusCode statusCode = HttpStatusCode.OK) where Output : IOutputModel

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/literal-life-church/live-streaming-api/pullRequest?prid=5573112)